extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -244,6 +244,10 @@
     }
 
     function trackBlocking(value) {
+        if (blockingTracked) {
```

Comment:
Not really, first `sp.blocking` is dispatched by detection script and when user is blocking then the second call is from recovery. Actually SP already fixed duplicated events in recovery :)